src/MCMCChains.jl

```diff
@@ -47,7 +47,11 @@ export mcse
 export rafterydiag
 export rstar
 
-export hpd
+# Reexport stats functions
+using PosteriorStats: SummaryStats, default_diagnostics, default_stats,
+    default_summary_stats, hdi, summarize
+export SummaryStats, default_diagnostics, default_stats, default_summary_stats, hdi,
+    summarize
```

Comment:
Should we actually reexport all of these functions? Based on the tests it seems
```suggestion
export SummaryStats, hdi, summarize
```
might be sufficient?

Reply:
It depends on how accessible we want these methods to be, as they offer different summary presets that I think will be useful to users. I also expect most users won't even know PosteriorStats exists so wouldn't know they exist otherwise. But I could also see the argument in leaving them out.